Cargo.toml

```diff
@@ -194,6 +194,8 @@ alloy-transport-http = { version = "0.1.2", default-features = false }
 alloy-transport-ipc = { version = "0.1.2", default-features = false }
 alloy-transport-ws = { version = "0.1.2", default-features = false }
 
+alloy-fork-db = { git = "https://github.com/klkvr/alloy-fork-db", rev = "10225cd" }
```

Comment:
rename to foundry-fork-db + release on crates.io?

Reply:
yep